examples/stats-verbose/README.md

```diff
@@ -0,0 +1,96 @@
+This configuration will enable the verbose output for the stats report, and it includes various options to display detailed information about the build process, such as:
+ 
+- [asset details](https://webpack.js.org/configuration/stats/#statsassets)
+- [chunk details](https://webpack.js.org/configuration/stats/#statschunks)
+- [module details](https://webpack.js.org/configuration/stats/#statsmodules)
+
+You see that everything is working nicely together.
+
+# example.js
+
+```javascript
+console.log("Hello World!");
+```
+
+# webpack.config.js
+
+```javascript
+const path = require("path");
+
+module.exports = {
+    entry: {
+		main: "./example"
+	},
+	output: {
+		path: path.join(__dirname, "dist"),
+		filename: "output.js",
+	},
+	stats: {
```

Comment:
Am I mistaken in that there is not a `options.stats="verbose"` preset in webpack which does this so the user doesn't have to set all these values to true?

Reply:
@TheLarkInn yes, but we want to disable logging as suggested in https://github.com/webpack/webpack/pull/16973#discussion_r1164323590